scene/gui/color_picker.cpp

```diff
@@ -2009,6 +2018,15 @@ ColorPicker::~ColorPicker() {
 
 /////////////////
 
+void ColorPickerPopupPanel::_input_from_window(const Ref<InputEvent> &p_event) {
+	if (get_flag(FLAG_POPUP) && p_event->is_action_pressed(SNAME("ui_accept"), false, true)) {
```

Comment:
Is the `get_flag()` check needed?

Reply:
I'm not sure, I think I just copied it over from a similar dialog and I left it in just in case. My guess was that it is a flag than can still be set separately regardless of the control being a PopupPanel, and thus you wouldn't want it to close automatically when you press enter if POPUP flag was set to false.

If this is an impossibility, we can remove it.

Reply:
Yeah it's impossible, in a way that nothing is setting this flag and it can't change by itself.